buddyboss/buddyboss-platform
# Fixed - Groups not display on the Group Type page

### All Submissions:






### Changes proposed in this Pull Request:

The Group ajax response comes with a 404 status code. I have added a status code in the `wp_send_json_success` function to get response correctly.


Fixes #2103  .

### How to test the changes in this Pull Request:

1. Create a Group Type
2. Assign Group Type to Groups
3. Go to the Group Type page ex: site.com/groups/type/{group-type-name}/
4. Groups should be displayed now

### Proof Screenshots or Video

https://www.loom.com/share/020215dfa5954c048f12795df3a705d1


### Other information:
